### PR TITLE
added language check before adding the tweet

### PR DESCRIPTION
**Pull request by piyshkmr**
Please review my PR and merge it.

### screenshot
Screenshot 2022-10-10 041405

### CODE REVIEW

**Comment by mr-palindrome** [comment]
Uncomment the else condition!

File: scrape.py [hunk 1]
```diff
@@ -42,18 +43,23 @@ def scrape(**kwargs):
...
+
+        if (language == "en"):
+            # bodies = ''.join(' ' + val.replace('\xa0', '').replace('\r\n', '').strip() for val in bodies)
+            tweets.append({
+                "id": str(tweet.id),
+                "url": tweet.url,
+                "content": bodies,
+                "date": str(tweet.date)
+            })
         # else:
```
Comment (mr-palindrome) [inline scrape.py]: remove the comment form the else statement as this will execute when the tweets are not in english.

**Comment by mr-palindrome** [comment]
Hey @devocoe!
I asked to uncomment the else part, if the tweet is not in english in that case the the total no. of tweets that is required will decrease. So, you need to add an else part where you will increase the max_count by 1 if the language is not english.

**Comment by piyshkmr** [comment]
> Hey @devocoe!
> I asked to uncomment the else part, if the tweet is not in english in that case the the total no. of tweets that is required will decrease. So, you need to add an else part where you will increase the max_count by 1 if the language is not english.

Ok i will fix that.

**Review by mr-palindrome — APPROVED** [review]

**Pull request merged**